assets/src/components/searchForm.tsx

```diff
@@ -257,64 +343,71 @@ export const SearchForm = ({
           >
             <CircleXIcon />
           </button>
-          <button
-            type="submit"
-            title="Submit"
-            className="c-search-form__submit"
-            onClick={onSubmit}
-            // TODO(design): add error states instead of using `disabled`
-            disabled={!isValidSearchText(inputText)}
-          >
-            <SearchIcon />
-          </button>
+          {comboboxType == "map_search" && (
+            <button
+              type="submit"
+              title="Submit"
+              className="c-search-form__submit"
+              onClick={onSubmit}
+              // TODO(design): add error states instead of using `disabled`
+              disabled={!isValidSearchText(inputText)}
+            >
+              <SearchIcon />
+            </button>
+          )}
+          {comboboxType == "select" && (
+            <button
+              // TODO: rename this class (follow-up PR)
+              type="button"
+              className="c-search-form__submit"
+              onClick={() => setAutocompleteEnabled(true)}
+            >
+              <ChevronDown />
+            </button>
+          )}
         </div>
         <div
           className="c-search-form__autocomplete-container"
           hidden={!autocompleteVisible}
         >
-          <SocketContext.Provider value={useSocket()}>
-            <GroupedAutocompleteFromSearchTextResults
-              id={autocompleteId}
-              controlName="Search Suggestions"
-              maxElementsPerGroup={5}
-              searchFilters={
-                property === "all"
-                  ? allFiltersOn
-                  : { ...allFiltersOff, [property]: true }
-              }
-              searchText={inputText}
-              fallbackOption={autocompleteOption(inputText, onSubmit)}
-              onSelectVehicleOption={onSelectVehicleOption}
-              onSelectedLocationId={onSelectedLocationId}
-              onSelectedLocationText={(text) => {
-                setAutocompleteEnabled(false)
-                onSelectedLocationText(text)
-              }}
-              controllerRef={autocompleteController}
-              onCursor={{
-                onCursorExitEdge: () => formSearchInput.current?.focus(),
-              }}
+          {comboboxType == "map_search" ? (
+            <MapSearchCombobox
+              autocompleteController={autocompleteController}
+              autocompleteId={autocompleteId}
+              dispatch={dispatch}
+              formSearchInput={formSearchInput}
+              onSubmit={onSubmit}
+              query={query}
+              setAutocompleteEnabled={setAutocompleteEnabled}
+            />
+          ) : (
+            <GrouplessAutocompleteFromArray
+              controlName=""
+              fallbackOption={autocompleteOption(
+                "No matching routes",
```

Comment:
nit(non-blocking): we chatted about this in a huddle, but it _may_ be beneficial if we rename this from `Combobox` to something about how this is a Map search OR route select _at this moment_. 

I'll post the combobox refactor ticket we talked about on this comment thread for posterity once it's created.